doc/api/http.markdown

```diff
@@ -966,6 +966,8 @@ Example:
 
     http.get("http://www.google.com/index.html", function(res) {
       console.log("Got response: " + res.statusCode);
+      // consume response body
+      res.resume();
```

Comment:
should this be above console.log as seen in @chrisdickinson's original [example?](https://github.com/nodejs/node-v0.x-archive/issues/8443#issuecomment-56762417)

might make sense to have the response enter flowing mode before accessing it.